tracks/javascript/exercises/collatz-conjecture/mentoring.md

```diff
@@ -0,0 +1,87 @@
+### Reasonable solution
+
+The collatz conjecture has multiple approaches when it comes to implementation,
+and both a recursive as an iterative implementation can be idiomatic.
```

Comment:
recursive `and` an iterative ?

Reply:
I wanted to say "as well as", but I like yours better.